doc/frameworks/pytorch/using_pytorch.rst

```diff
@@ -292,7 +292,104 @@ using two ``ml.p4d.24xlarge`` instances:
 
     pt_estimator.fit("s3://bucket/path/to/training/data")
 
+.. _distributed-pytorch-training-on-trainium:
+=============================================
```

Comment:
Need to bring the title back. `.. _distributed-pytorch-training-on-trainium:` is only a flag that enables links.